scripts/deploy/main.bicep

```diff
@@ -14,7 +14,12 @@ param webAppServiceSku string = 'B1'
 
 @description('Location of package to deploy as the web service')
 #disable-next-line no-hardcoded-env-urls
```

Comment:
When you update main.bicep you need to also build a new version of main.json from it. You can do it from vscode if you have the bicep extension, or from az cli.